docs/src/main/asciidoc/picocli.adoc

```diff
@@ -291,6 +291,81 @@ annotationProcessor 'info.picocli:picocli-codegen'
 
 In the development mode, i.e. when running `mvn quarkus:dev`, the application is executed and restarted every time the `Space bar` key is pressed. You can also pass arguments to your command line app via the `quarkus.args` system property, e.g. `mvn quarkus:dev -Dquarkus.args='--help'` and `mvn quarkus:dev -Dquarkus.args='-c -w --val 1'`.
 
+== Kubernetes support
+
+Once you have your command line application, you can also generate the resources necessary to install and use this application in Kubernetes by adding the `kubernetes` extension. To install the `kubernetes` extension, run the following command in your project base directory.
+
+[source,bash]
+----
+./mvnw quarkus:add-extension -Dextensions="kubernetes"
+----
+
+This will add the following to your `pom.xml`:
+
+[source,xml]
+----
+<dependency>
+    <groupId>io.quarkus</groupId>
+    <artifactId>quarkus-kubernetes</artifactId>
+</dependency>
+----
```

Comment:
Ideally you should use the tab thing here to have Maven/CLI and Gradle/Maven alternatives. See other guides for examples.

Reply:
Awesome! I also changed the other one in the same guide for picocli.